pkgs/development/python-modules/mujoco/default.nix

```diff
@@ -63,6 +71,7 @@ buildPythonPackage rec {
     description =
       "Python bindings for MuJoCo: a general purpose physics simulator.";
     homepage = "https://mujoco.org/";
+    changelog = "https://github.com/google-deepmind/mujoco/releases/tag/${version}";
```

Comment:
Might as well add this link to the C library while we are at it.

Reply:
Yes, good point.